server/plugin.go

```diff
@@ -878,19 +880,29 @@ func (p *Plugin) GetRemoteID() string {
 }
 
 func (p *Plugin) updateMetrics() {
-	start := time.Now()
-	p.API.LogDebug("Updating metrics")
-	stats, err := p.store.GetStats(p.remoteID, PreferenceCategoryPlugin)
+	now := time.Now()
+	p.API.LogInfo("Updating metrics")
+
+	stats, err := p.store.GetStats(storemodels.GetStatsOptions{
+		RemoteID:           p.remoteID,
+		PreferenceCategory: PreferenceCategoryPlugin,
+		ActiveUsersFrom:    now.Add(-metricsActiveUsersRange),
+		ActiveUsersTo:      now,
```

Comment:
This parameter feels incorrect: is it actually meaningful to query to anything other than `ActiveUsersTo == now`?